runner/output.go

```diff
@@ -0,0 +1,24 @@
+package runner
+
+import (
+	"io"
+)
+
+// Saver lets clients create new Outputs so they can save data.
+// This is how Runner can save stdout and stderr.
+// (NB: Saver is temporary until we save output into a new Snapshot)
+type Saver interface {
```

Comment:
As discussed in-person:

Saver:FileSystem::Output:File

Renamed Saver -> OutputCreator